client/src/components/Home/AppIntro.jsx

```diff
@@ -4,6 +4,13 @@ import PropTypes from 'prop-types';
 
 import efo from './../../../images/efo.jpg';
 
+/**
+ * Stateless component for rendering intro banner
+ * on landing page
+ *
+ * @param {object} props
+ * @returns {view} AppIntro
```

Comment:
Work on all the jsdocs across the whole application. Ensure it is descriptive this includes the params, what is returned etc

Reply:
`/**
 * @description handles create a new group
 *
 * @param { object } group - contains group details
 *
 * @return { object } group - returns create group success-failure action
 *
 */` let this be your guid

Reply:
Thanks, I will do that